README.md

```diff
@@ -1,59 +1,83 @@
-> #### Additionally, quickstart can be used to showcase flows with MFA
-> By default, MFA is turned off.
-> 1. Run `make enable-mfa` to turn MFA on.
-> 2. Run `make disable-mfa` to turn MFA off.
+#### Required software
 
-<br/>
-<h3 id="makefile-targets"> Makefile Targets</h3>
+- `make`
+- `docker` > 20.10.2
+- `docker-compose` > 1.29.0
 
-The current types of runtimes are currently supported: 
-1. Open Banking UK with local ACP instance: `make run-obuk-local`
-2. Open Banking UK with SAAS ACP: `make run-obuk-saas`
-3. Open Banking Brasil with local ACP instance: `make run-obbr-local`
-4. Open Banking Brasil with SAAS ACP: `make run-obbr-saas`
-5. CDR with local ACP instance: `make run-cdr-local`
+For Windows you need to use WSL.
 
-<br/>
+#### Deployment mode
 
-## How to run Cypress Tests
-| Mode      | Instructions |
-| ----------- | ----------- |
-| With Cypress UI      | Run `make run-tests` to open Cypress GUI with tests scenarios. Then click on the intended test suite name `{TEST_NAME}.ts`    |
-| Headless (commandline)   | Run one of the following: `run-obuk-tests-headless`, `run-obbr-tests-headless`, `run-cdr-tests-headless`        |
+There are two ways you can run quickstart.
+
+If you want to set up everything locally then you need to contact sales@cloudentity.com to get credentials to be able to download ACP docker image.
+
+The other option is to register and use ACP in SaaS: https://authz.cloudentity.io
 
-<br/>
+| ACP type      | Associated Environment File | Additional Configuration needed?
+| ----------- | ----------- | --------|
+| Local      | .env-local   | No, everything is good to go out of the box. You shouldn't need to touch this file|
+| SAAS   | .env-saas        | <a href="#saas-configuration-instructions">Yes, running with saas takes a few extra steps</a>|
 
 <h2 id="saas-configuration-instructions">Configuring your ACP SAAS Tenant For Quickstart</h3>
 
 1. Enable System workspace on your ACP SAAS Tenant (you will most likely need to contact someone at Cloudentity to do this for you)
 2. Go to "System" workspace and create a new application with `manage_configuration` scope and `client_credentials` grant type
 3. Configure the `.env-saas` file with the following:
     * Saas tenant id
-    * id of the client you created in step 2 
+    * id of the client you created in step 2
     * secret of the client you created in step 2
 4. Now you are ready to run one of the saas related <a href="#makefile-targets"> makefile targets</a>
 
-<br/>
+## How to Run Quickstart
+
+The current types of runtimes are currently supported:
+
+1. Open Banking UK with local ACP instance: `make run-obuk-local`
+2. Open Banking UK with SAAS ACP: `make run-obuk-saas`
+3. Open Banking Brasil with local ACP instance: `make run-obbr-local`
+4. Open Banking Brasil with SAAS ACP: `make run-obbr-saas`
+5. CDR with local ACP instance: `make run-cdr-local`
+6. FDX with local ACP instance: `make run-fdx-local`
+7. FDX with SAAS ACP: `make run-fdx-saas`
+
```

Comment:
What about `make clean` operation?
Is this intentional to not describe it?
Currently, if user will run `make run-obbr-local` and after that will try to switch to UK spec via `make run-obuk-local` operation without executing `make clean`, then CLI command will stuck due to:

> docker ps -a | grep configuration
5c44f9d9b0c0   cloudentity/openbanking-quickstart-configuration:latest                 "/app/main --tenant-…"   6 minutes ago   Restarting (1) 54 seconds ago

Reply:
Added